presto-main/src/test/java/com/facebook/presto/type/TestMapOperators.java

```diff
@@ -860,6 +861,40 @@ public void testMapFromEntries()
         assertCachedInstanceHasBoundedRetainedSize("map_from_entries(ARRAY[('a', 1.0), ('b', 2.0), ('c', 3.0), ('d', 4.0), ('e', 5.0), ('f', 6.0)])");
     }
 
+    @Test
+    public void testMultimapFromEntries()
```

Comment:
I ran a few simple tests and `select multimap_from_entries(ARRAY[null]);` fails with an IAE.